Dockerfile

```diff
@@ -52,13 +48,17 @@ RUN curl --location https://github.com/kubernetes-sigs/kustomize/releases/downlo
         chmod +x /usr/local/bin/kustomize
 
 # Install Google Cloud SDK
-RUN curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
-        tar xzf google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
-        rm google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
+RUN curl --location https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz --output google-cloud-sdk.tar.gz && \
```

Comment:
You've changed the `curl` exectuion to be consistent, right?  If so, please, update also the installation of `yq` which uses `wget` atm.